package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#task/ECEP-113-support-news",
```

Comment:
**⚠️ Warning!** This change should **not** be committed. It is required only for testing (remote and local).